ramls/loan.json

```diff
@@ -93,6 +93,11 @@
       "description": "Indicates whether or not this loan had its due date modified by a recall on the loaned item",
       "type": "boolean"
     },
+    "declaredLostDate" : {
```

Comment:
I believe there was also a discussion on the issue about a comment about why an item was declared lost, is that going to be implemented as well?

Reply:
Storing comments is already implemented using the loan history mechanism.

Reply:
Ok, so the intention is to use the `last action comment` for declared lost comments?

Reply:
As per the clarificaitons from the PO, `the comments appear in that loan action table. ` - that is the loan history. I don't think that we need to store the last comment specifically.

Reply:
Ok, thank you for explaining that. I'll resolve this.

(Personally I think the PO is likely describing the behaviour from a user experience perspective where the loan action table in the UI that happens to be constructed from the loan history)